WpfPlus/Styles/TabControlStyle.xaml

```diff
@@ -52,15 +62,27 @@
             <Setter.Value>
                 <ControlTemplate TargetType="{x:Type TabControl}">
                     <Grid KeyboardNavigation.TabNavigation="Local">
+                        
```

Comment:
Please remove this and following empty lines

Reply:
Blank lines removed.